packages/yswaps/scripts/execute-mainnet-pending-trades.ts

```diff
@@ -28,92 +28,67 @@ let flashbotsProvider: FlashbotsBundleProvider;
 
 type FlashbotBundle = Array<FlashbotsBundleTransaction | FlashbotsBundleRawTransaction>;
 
-type PendingTrade = [BigNumber, string, string, string, BigNumber, BigNumber] & {
-  _id: BigNumber;
-  _strategy: string;
-  _tokenIn: string;
-  _tokenOut: string;
-  _amountIn: BigNumber;
-  _deadline: BigNumber;
-};
-
-type TradeSetup = {
-  swapper: string;
-  data: string;
-  minAmountOut: BigNumber | undefined;
-};
-
 async function main() {
   const chainId = await getChainId();
   console.log('[Setup] Chain ID:', chainId);
+
   const [signer] = await ethers.getSigners();
+  const multicalls = [new ThreePoolCrvMulticall()];
+
   console.log('[Setup] Creating flashbots provider ...');
   flashbotsProvider = await FlashbotsBundleProvider.create(
     ethers.provider, // a normal ethers.js provider, to perform gas estimiations and nonce lookups
     signer // ethers.js signer wallet, only for signing request payloads, not transactions
   );
 
-  const tradeFactory = await ethers.getContract<TradeFactory>('TradeFactory');
+  // env setup for testing. Should not be necessary
+  const strategy: Signer = await impersonate('0xa48c616144FD4429b216A86388CAb0Eed990cE87');
+  const ymech: Signer = await impersonate('0x2C01B4AD51a67E2d8F02208F54dF9aC4c0B778B6');
+
+  let tf: TradeFactoryExecutor = TradeFactoryExecutor__factory.connect('0xBf26Ff7C7367ee7075443c4F95dEeeE77432614d', ymech);
```

Comment:
```suggestion
  let tradeFactory: TradeFactoryExecutor = TradeFactoryExecutor__factory.connect('0xBf26Ff7C7367ee7075443c4F95dEeeE77432614d', ymech);
```
please use descriptive names